migrations/versions/0336_broadcast_msg_migrate_content.py

```diff
@@ -0,0 +1,33 @@
+"""
+
+Revision ID: 0336_broadcast_msg_content_2
```

Comment:
The Revision ID doesn't match the filename, which I think will cause issues with the script to check if migrations need to run

Reply:
ah good spot 👍